docs/installation.rst

```diff
@@ -9,14 +9,18 @@ you may want to use trusted certificate and other enhancement.
 Prerequisites
 =============
 
-In order to run temBoard agent, you require:
+In order to run temBoard agent, you need:
 
-- PostgreSQL 9.4+, listening on UNIX socket, on the same host. Check with ``sudo -u postgres psql``.
+- PostgreSQL 9.4+, running on Linux and listening on UNIX socket. Check with ``sudo -u postgres psql``.
```

Comment:
Worth a dedicated entry for OS.

Reply:
done in 2bd89a41c6daec31496b604bba5e2007485dadd5